x-team/xp-ui
# Adjust font-size of Jobs Dashboard components

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3497

## Description

Adjust font-size of Jobs Dashboard components.

Attention during review: the majority of the text elements doesn't have a representation for the mobile version in Figma. I made adjustments from my gut feeling trying to preserve harmony and somehow a consistency with the other text elements that exists in Figma. 

## Checklist


## Steps to Test or Reproduce

Check mobile viewport for the components:

- http://localhost:9001/?path=/story/ui-components-closedjobapplications--basic-usage
- http://localhost:9001/?path=/story/ui-components-headerbar-use-cases--with-links
- http://localhost:9001/?path=/story/ui-components-jobapplicationcard--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobcard--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobdetails--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobskills--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobspagebreadcrumbs--basic-usage
- http://localhost:9001/?path=/story/ui-components-welcomehero--basic-usage
- http://localhost:9001/?path=/story/ui-components-emptystate--basic-usage
- http://localhost:9001/?path=/story/ui-components-applicantscreennotification--basic-usage
- http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout--basic-usage
- http://localhost:9001/?path=/story/core-components-confirmationbox--basic-usage
- http://localhost:9001/?path=/story/core-components-errorbox--basic-usage


## Impacted Areas in Application

- `ClosedJobApplications`
- `HeaderBar`
- `JobApplicationCard`
- `JobCard`
- `JobDetails`
- `JobSkills`
- `JobsPageBreadcrumbs`
- `WelcomeHero`
- `EmptyState`
- `ApplicantScreenNotification`
- `JobsPageLayout`
- `ConfirmationBox`
- `ErrorBox`
